CBI.Jenkinsfile

```diff
@@ -140,22 +140,20 @@ spec:
       script {
         def envName = ''
         if (env.JENKINS_URL.contains('ci.eclipse.org/xtext')) {
-          envName = ' (JIPP)'
+          envName = ' (JIRO)'
```

Comment:
intended?

Reply:
yes, as stated in the issue:
CBI.Jenkinsfile is not active on the current JIPP. I expect that ci-staging becomes ci after the server has been migrated to JIRO.